spec.emu

```diff
@@ -379,7 +422,7 @@ contributors: Google, Ecma International
         1. Let _code_ be ? ToString(_code_).
         1. If the result of ! IsValidCodeForDisplayNames(_displayNames_.[[Type]], _code_) is *false*, throw a *RangeError* exception.
         1. Let _fields_ be _displayNames_.[[Fields]].
-        1. Let _name_ be _fields_[[_code_]].
+        1. Let _name_ be _fields_[[&lt;_code_&gt;]].
```

Comment:
is the slot name `<code>`?

Reply:
no, the variable is code, as defined in "Let _code_ be ? ToString(_code_) 3 steps above.

Reply:
ah, it's a dynamic lookup. Is that done anywhere else? Typically slot names are exclusively static/hardcoded.

Reply:
It's a common idiom in `[[LocaleData]]` slots in ECMA-402.